### DISCUSSION TITLE
InfluxDB "could not sign in"

### DISCUSSION THREAD

**Discussion by ByteAndBits** [Security-Onion-Solutions/securityonion#13917] [2.4] [answered]
### Version

2.4.110

### Installation Method

Security Onion ISO image

### Description

other (please provide detail below)

### Installation Type

Distributed

### Location

on-prem with Internet access

### Hardware Specs

Exceeds minimum requirements

### CPU

64

### RAM

1TB

### Storage for /

35TB

### Storage for /nsm

70TB

### Network Traffic Collection

span port

### Network Traffic Speeds

1Gbps to 10Gbps

### Status

Yes, all services on all nodes are running OK

### Salt Status

No, there are no failures

### Logs

No, there are no additional clues

### Detail

Hi,

we can´t login to influxdb.
1. we activated MFA for users
2. we created users and ad to superuser
3. all users are with MFA
4. we delete the first admin user who created with initial installation

Is here something broken?

### Guidelines

- [X] I have read the discussion guidelines at https://github.com/Security-Onion-Solutions/securityonion/discussions/1720 and assert that I have followed the guidelines.

**Comment by InfosecGoon** [comment, marked as answer]
Reset a user's password using the Administration --> Users interface (NOT the self-service password reset for the user) and see if it works.
